Tests/Tests/STPBinRangeTest.m

```diff
@@ -95,11 +132,11 @@ - (void)testBinRangesForBrand {
     }
 }
 
-- (void)testMostSpecifiBinRangeForNumber {
+- (void)testMostSpecificBinRangeForNumber {
     STPBINRange *binRange;
     
     binRange = [STPBINRange mostSpecificBINRangeForNumber:@""];
-    XCTAssertEqual(binRange.brand, STPCardBrandUnknown);
+    XCTAssertNotEqual(binRange.brand, STPCardBrandUnknown);
```

Comment:
FYI I am not even sure this test makes sense and may remove it? The most specific bin range for no string could be anything. Also the returned value for mostSpecific when more than one possible range has the same specific-ness is undefined.